PendingReleaseNotes.md

```diff
@@ -6,6 +6,9 @@
 - The `fsType` default for StorageClass examples are now using XFS to bring it in line with Ceph recommendations.
 - Ceph is updated from Luminous 12.2.5 to 12.2.7.
 - Ceph OSDs will be automatically updated by the operator when there is a change to the operator version or when the OSD configuration changes. See the [OSD upgrade notes](Documentation/upgrade-patch.md#object-storage-daemons-osds).
+- Rook Ceph block storage provisioner can now correctly create erasure coded block images.
+    - For this to work you need to create two Ceph Pools in total, one replicated (which is used for metadata of the block images) and one erasure coded (which is used for data of the block images).
```

Comment:
Should these details be moved to the doc that describes the advanced scenario? In general, a single line of release notes per feature seems good, with a link to more details.